src/main/java/org/isf/medicalstock/model/Lot.java

```diff
@@ -84,15 +84,50 @@
 	@Column(name="LT_COST")
 	private BigDecimal cost;
 
+	/**
+	 * Automatic calculated field for a lot's quantity stocked in the main store, 
+	 * taking in account only the main MedicalStock movements (charges and discharges).<br/>
```

Comment:
DocLint reports that self-closing element is not allowed.   I think just replace it with `<br>`

Reply:
Done